Classes/Authentication/Common/AccountType/ViewModels/AccountTypeViewModel.swift

```diff
@@ -24,11 +24,19 @@ struct AccountTypeViewModel: PairedViewModel {
     private(set) var title: EditText?
     private(set) var detail: EditText?
     private(set) var badge: String?
+    private(set) var shouldShowNewAccountWarning: Bool = false
     
     init(_ model: AccountSetupMode) {
         bindImage(model)
         bindTitle(model)
         bindDetail(model)
+        
+        switch model {
```

Comment:
This can potentially can go to the AccountSetupMode extension:
```swift
extension AccountSetupMode {

    var shouldShowNewAccountWarning: Bool {
        switch self {
            case .addBip39Address:
                return true
            case {Other cases}:
                return false
        }
    }
}
```